articles/getting-started-with-game-development-using-windows-forms/index.md

```diff
@@ -0,0 +1,320 @@
+it will reset the counter (i) and show the colors again on each click.
+```c#
+ int i = 0;
+        void Changecolor(Button x)
+        {
+            i++;
+            if (i == 1)
+                x.BackColor = Color.Blue;
+            if (i == 2)
+                x.BackColor = Color.Red;
+            if (i == 3)
+                x.BackColor = Color.Green;
+            if (i == 4)
+                x.BackColor = Color.Yellow;
+            if (i == 5)
+                x.BackColor = Color.Pink;
+            if (i == 6)
+                x.BackColor = Color.Purple;
+            if (i == 7)
+                i = 0;
+          }
+```
+On each (Col1,Col2,Col3,...) button we double click on it and use this method on its button instead of x.
+
+```c#
+private void Col1_Click(object sender, EventArgs e)
+        {
+            Changecolor(Col1);
+        }
+private void Col2_Click(object sender, EventArgs e)
+        {
+            Changecolor(Col2);
+        }
+        
+ // Repeat for all the buttons ...
+ 
+ private void Col40_Click(object sender, EventArgs e)
+        {
+            Changecolor(Col40);
+        }
+```
+Now we need to define the GO method that will start the game and produce the 4 random colors.
+First we define a random object from the random class, then we use it to produce 4 random numbers from 1 to 6, and each color will have a value.
+
+```c#
+ private void GO_Click(object sender, EventArgs e)
+        {
+            Random rnd = new Random();
+            int a = rnd.Next(1, 7);
+            int b = rnd.Next(1, 7);
+            int c = rnd.Next(1, 7);
+            int d = rnd.Next(1, 7);
+            int blue = 1, red = 2, green = 3, yellow = 4, pink = 5, purple = 6;
+```
+After that, we need to check what the random method produced for each digit, and therefore we need to check the `a` 6 times for each color.
+
+```c#
+ if (a == blue)
+                cool1.BackColor = Color.Blue;
+            if (a == red)
+                cool1.BackColor = Color.Red;
+            if (a == green)
+                cool1.BackColor = Color.Green;
+            if (a == yellow)
+                cool1.BackColor = Color.Yellow;
+            if (a == pink)
+                cool1.BackColor = Color.Pink;
+            if (a == purple)
+                cool1.BackColor = Color.Purple;              
+ //Repeat this with the snippet with the rest of the digits b c d.   
+ 
+ ch1.Visible = true;
+ //The previous line will turn the first Check button to Visible in order to start the first round.
+        }        
+```
+**NOTE** that you need to repeat the previous piece of code with `b` `c` and `d` before closing the method.
+
+
+Now we need to define the Check method that will check the colors chosed by the player and compare them with the colors produced by the system.
+The method will take the 4 colors chosed by the player the 4 colors produced by the system and the 4 hint digits as parameters.

```

Comment:
replace chosed with chosen i.e

Now we need to define the Check method that will check the colors chosen by the player and compare them with the colors produced by the system.